documentation/README.md

```diff
@@ -65,4 +65,27 @@ Some of the guiding principles for this aproach are:
 * We must avoid burying test parameters into the test scripts. This allows the requirements to be maintained/defined independently from the (often complex) code which executes tests. 
 * Many projects exist where only a programmer can locate and ferret out actual test criteria, often expressed as hard-coded constants buried within thousands of lines of test automation code. For quality control, these criteria must be easily accessible, reviewable and maintainable, to anyone familiar with the project.
 * We advocate complete auditability and traceability of tests cases, test results, associated specs and DUT/SUT configuration. This means a test run will record versions of every item including GitHub Repo commit SHA ids, branches, tags, SW versions, API versions, etc.
-* Clear, concise and to the point human-readable reports, plus machine-readable results allowing dashboards, rolling-up of results, etc.
\ No newline at end of file
+* Clear, concise and to the point human-readable reports, plus machine-readable results allowing dashboards, rolling-up of results, etc.
+
+## Repositories
+
+- [SONiC](https://github.com/Azure/SONiC)
+- [SAI](https://github.com/opencomputeproject/SAI)
+- [P4](https://opennetworking.org/p4) and [P4 working group](https://p4.org/working-groups)
+- [PINS](opennetworking.org/pins)
+- [PNA Consortium Spec](https://p4.org/p4-spec/docs/PNA-v0.5.0.html)
+- [IPDK](https://ipdk.io/) and [IPDK GitHub](https://github.com/ipdk-io/ipdk-io.github.io)
+- [bmv2 - behavioral model v2](https://github.com/p4lang/behavioral-model)
+- [DPDK](https://www.dpdk.org)
+
+
+## References
+
+- [Glossary](https://github.com/Azure/DASH/wiki/Glossary)
+- [FAQ](https://github.com/Azure/DASH/wiki/FAQ)
+- [SAI](./sai/README.md)
```

Comment:
Please remove everything from here down - they're internal links which are already stale and redundant with other ToCs we've created. Thx.

Reply:
Removed the links.

Reply:
Links are still visible???

Reply:
It helps if I commit the changes. 🌞 
Done.